### PR TITLE
Simplify sys params to closer match reality

### LINKED ISSUE #629: Remove duplicated system parameters file inputs
### Expected Behavior
Single input fields for 5G central pump design head and flow rate

### Actual Behavior
Potentially duplicated input fields. The ETS [`pump_design_head`](https://github.com/urbanopt/geojson-modelica-translator/blob/11899554adb23be3d125efdff91ca31b9692f74e/geojson_modelica_translator/system_parameters/schema.json#L1301) under ETS definitions might be a duplicate of the central [`pump_design_head`](https://github.com/urbanopt/geojson-modelica-translator/blob/11899554adb23be3d125efdff91ca31b9692f74e/geojson_modelica_translator/system_parameters/schema.json#L1256). Similarly, the ETS [`pump_flow_rate`](https://github.com/urbanopt/geojson-modelica-translator/blob/11899554adb23be3d125efdff91ca31b9692f74e/geojson_modelica_translator/system_parameters/schema.json#L1296) should be put under central pump definitions.

### Version Information

Modelica Buildings Library: `X.Y.Z`
Modelica Standard Library: `X.Y.Z`
TEASER: `X.Y.Z`
Python: `X.Y.Z`

### PR DESCRIPTION
**Pull request by vtnate**
#### Any background context you want to provide?
We developed the schema to cover many potential 5G parameters. This cleans up a few of them, to reduce duplication and have parameters where they make sense.

This says 154 commits but that is because it was originally based off the multi-ghe PR, so Github thinks those 140 commits are a part of this PR. It's not that big, just changing schema and test sys-param files to match.
#### What does this PR accomplish?
- Move `pump_design_head` & `pump_flow_rate` to district params instead of individual ETS params.
- Spread out district param definitions in the schema
- Use more precise validation in several instances
- Require ETS for all buildings
- Update a few tests to match new schema
#### How should this be manually tested?
CI is technically efficient. Manual review to determine if this is the appropriate place for them to be.
#### What are the relevant tickets?
Resolves #629

### CODE REVIEW

File: tests/data_shared/system_params_des_5g.json [hunk 1]
```diff
@@ -109,11 +103,24 @@
...
-      "central_heating_plant_parameters": {
-        "temp_setpoint_hhw": 54
+      "soil": {
+        "conductivity": 2.0,
+        "rho_cp": 2343493,
+        "undisturbed_temp": 18.3
+      },
+      "central_pump_parameters": {
+        "pump_design_head": 60000,
+        "pump_flow_rate": 0.01
```
Comment (JingWang-CUB) [inline tests/data_shared/system_params_des_5g.json:123]: ThermalNetwork should be aware of these changes for its sizing of GHEs.

File: geojson_modelica_translator/system_parameters/time_series_template.json [hunk 2]
```diff
@@ -118,7 +116,8 @@
         "chp_installed": false
       },
       "central_pump_parameters": {
-        "pump_design_head": 60000
+        "pump_design_head": 60000,
+        "pump_flow_rate": 0.01
```
[the remark this answers is not in the document]
Comment (JingWang-CUB) [reply geojson_modelica_translator/system_parameters/time_series_template.json:120]: In other words, these should be 5G only changes.

**Review by JingWang-CUB — CHANGES_REQUESTED** [review]
Hi @vtnate , please see the detailed review comments. Otherwise, it looks good!

File: tests/data_shared/system_params_des_5g.json (hunk 1 above)
Comment (vtnate) [reply tests/data_shared/system_params_des_5g.json:123]: I have an open PR on TN doing just that: https://github.com/NREL/ThermalNetwork/pull/37.

File: geojson_modelica_translator/system_parameters/time_series_template.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (vtnate) [reply geojson_modelica_translator/system_parameters/time_series_template.json:120]: Should the entire `central_pump_parameters` block be removed from 4G districts?

We still want to remove the head & flow-rate from all ETS parameters, correct (as I did in this PR)?

[the remark this answers is not in the document]
Comment (JingWang-CUB) [reply geojson_modelica_translator/system_parameters/time_series_template.json:120]: To your first question, yes. Can you provide an example of your second question?

[the remark this answers is not in the document]
Comment (JingWang-CUB) [reply geojson_modelica_translator/system_parameters/time_series_template.json]: Yes, deleting the `pump_flow_rate` and `pump_design_head` from the 5g ETS parameters is appropriate.

**Review by JingWang-CUB — APPROVED** [review]

File: geojson_modelica_translator/system_parameters/schema.json [hunk 3]
```diff
@@ -217,7 +217,7 @@
       "type": "object",
       "properties": {
         "geojson_id": {
-          "description": "The GeoJSON ID as defined in the GeoJSON file. This is used to overwrite the default data for a specific building. This option is not used in the default section.",
+          "description": "The GeoJSON ID as defined in the GeoJSON file.",
```
Comment (nllong) [inline geojson_modelica_translator/system_parameters/schema.json:220]: nice cleanup

**Review by nllong — APPROVED** [review]
we are probably need to start versioning this file soon... 

changes look swell.

**Pull request merged**